website/source/docs/agent/options.html.md

```diff
@@ -1329,6 +1329,14 @@ Consul will not enable TLS for the HTTP API unless the `https` port has been ass
       is overlap between two rules, the more specific rule will take precedence. Blocking will take priority if the same
       prefix is listed multiple times.
 
+    * <a name="telemetry-prometheus_retention_time"></a><a href="telemetry-prometheus_retention_time">prometheus_retention_time</a>
```

Comment:
maybe you should give a recommended value (it seems that setting a multiple of the prometheus scrape interval is necessary to have meaningful counters)

Reply:
Yes, updating doc in the next patch